src/Jellyfin.Database/Jellyfin.Database.Providers.SqLite/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,23 @@
+using System.Reflection;
+using System.Resources;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Jellyfin.Database.Providers.SqLite")]
```

Comment:
There's some slight inconsistency in capitalization. The real capitalization is `SQLite`, but the namespace (and folder) is `SqLite`, but some of the files are `Sqlite`. Either is fine with me, but we should stick to one.

Reply:
Well SQLite is the technical correct name i think and gets used within abitrary strings whereever possible but due to the naming scheme we use, i think we only capitalise words even if the original name would be all capitalised.

So all userfacing instances should be "SQLite" but actual code uses "SqLite". I am fine with both too, the nuget package is also called `Sqlite` i think to prevent just that but comments above from others prefered "SQLite" in free texts and "Sqlite" in code as its in line with the source package.